docs-md/plugins/index.md

```diff
@@ -73,6 +77,85 @@ export interface EchoPlugin {
 }
 ```
 
+To implement new functionality in your plugin, begin by defining it in the exported interface:
```

Comment:
Original conversation from the OG PR:

![image](https://user-images.githubusercontent.com/2545368/84074009-76697500-a997-11ea-8277-ba7dd8c7ec20.png)